subscription_oca/models/sale_subscription.py

```diff
@@ -272,8 +274,21 @@ def _prepare_sale_order(self, line_ids=False):
             "order_line": line_ids,
         }
 
+    def _generate_subscription_date_range(self):
+        self.ensure_one()
+        if self.account_invoice_ids_count == 0:
+            start_date = self.date_start
+            end_date = self.recurring_next_date
```

Comment:
Shouldn't this be ```self._calculate_recurring_next_date(self.date_start)```
Now the period of the first invoice is:
![image](https://github.com/user-attachments/assets/7cadbaef-247b-4aa9-9aca-426b3c9c72d5)
But the next is 
![image](https://github.com/user-attachments/assets/83b58030-de5b-4a21-b5ad-451367f378d8)

I would expect the first to be 04/17/2025 and 05/17/2025